modules/apps/foundation/user-associated-data/user-associated-data-web/bnd.bnd

```diff
@@ -1,3 +1,6 @@
 Bundle-Name: Liferay User Associated Data Web
 Bundle-SymbolicName: com.liferay.user.associated.data.web
-Bundle-Version: 1.0.0
\ No newline at end of file
+Bundle-Version: 1.0.0
+Export-Package:\
```

Comment:
Since we are exposing an API I don't think those packages should go in a -web module (which is intended only to expose a UI), but rather in an -api module

Reply:
Never mind, I've seen you do this in a later commit.